clients/tabby-chat-panel/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tabby-chat-panel",
   "type": "module",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
As we have a back-compatiple update in the tabby-chat-panel api, let's bump its version.